src/shapes/group.class.js

```diff
@@ -131,7 +131,7 @@
     _updateObjectCoords: function(object, center) {
       var objectLeft = object.left,
           objectTop = object.top,
-          skipControls = true;
+          skipControls = false;
```

Comment:
i think the whoe reason of skipControls was this one, probably it can be removed alltogether

Reply:
I thought so but was afraid to break.
I will remove.

Reply:
bdbca70d

Reply:
I saw it's used only by StaticCanvas background/overlay Image